armi/physics/neutronics/fissionProductModel/tests/test_lumpedFissionProduct.py

```diff
@@ -233,59 +227,6 @@ def test_getAllFissionProductNames(self):
         self.assertAlmostEqual(self.lfps["LFP35"].getTotalYield(), 2.0)
 
 
-class TestExpandCollapse(unittest.TestCase):
-    """Test of the ability of the fission product file to expand from the LFPs"""
-
-    def test_expand(self):
-
-        fpd = lumpedFissionProduct.FissionProductDefinitionFile(io.StringIO(LFP_TEXT))
-        lfps = fpd.createSingleLFPCollectionFromFile("LFP35")
-
-        massFrac = {
-            "U238": 24482008.501781337,
-            "LFP35": 0.0,
-            "CL35": 1617.0794376133247,
-            "CL37": 17091083.486970097,
-            "U235": 3390057.9136671578,
-            "NA23": 367662.29994516558,
-        }
```

Comment:
Collapsing lumped fission products into a single lumped fission product during a run was never used and this was just stale/dead code. I don't think there is any reason to keep it around.